docs/usage-guide.md

```diff
@@ -0,0 +1,85 @@
+# Usage Guide for Osmo-Notify
```

Comment:
Add this in readme just like other 2 documents.

Reply:
Added